config/examples/cdc_replication.yaml

```diff
@@ -0,0 +1,54 @@
+input:
+  postgres_cdc:
+    dsn: postgres://me:foobar@localhost:5432?sslmode=disable
+    include_transaction_markers: true
+    slot_name: test_slot_native_decoder
+    stream_snapshot: true
+    schema: public
+    tables: [my_src_table]
+    # Group by transaction, each message batch is all rows changed in a transaction
+    # this might be massive, but might be required for foreign key constraints
+    batching:
+      check: '@operation == "commit"'
+      period: 10s
```

Comment:
This might need a disclaimer that it needs to be large enough to account for massive transactions... Or maybe the `mapping` processor below should use batch-wide queries (`from_all().<insert sanity check here>` when `batch_index() == 0`) to determine if a transaction looks incomplete and throw an error?

Reply:
Added a disclaimer! Thanks ❤️